webrender_api/src/display_list.rs

```diff
@@ -1162,6 +1162,7 @@ impl DisplayListBuilder {
         complex_clips: I,
         image_mask: Option<ImageMask>,
         scroll_sensitivity: ScrollSensitivity,
+        enable_scrollbars: bool
```

Comment:
I think it would be better to use an enum for this argument. Just judging by the changes to the examples, it's already a little hard to tell what this argument does. I suggest something like

```
enum IframeScrollbars {
    Enabled,
    Disabled,
}